services/migrations/gitlab.go

```diff
@@ -488,30 +489,8 @@ func (g *GitlabDownloader) GetComments(commentable base.Commentable) ([]*base.Co
 			return nil, false, fmt.Errorf("error while listing comments: %v %w", g.repoID, err)
 		}
 		for _, comment := range comments {
-			// Flatten comment threads
-			if !comment.IndividualNote {
```

Comment:
This line now will be ignored?

Reply:
Both branches here were exactly the same, except only the first note was converted if `IndividualNote` is true. That isn't really necessary, because `IndividualNote` means that this is not a discussion (=thread) but just a single comment that cannot be responded to. There will always be exactly one note in such a "discussion", so there is no reason to explicitly ignore any further notes.